### ISSUE TITLE
Consider adding an "On Hold" label

### ISSUE THREAD

**Issue by jsf9k** [cisagov/.github#10]
# 🚀 Feature Proposal #

We should consider adding an "On Hold" label that is distinct from the "Blocked" label.  See the discussion in cisagov/skeleton-generic#63.  It may even make sense to create both an "On Hold" label and a more specific "Waiting for Kraken" label.

## Motivation ##

When we are holding back a PR because we are waiting to gather enough PRs for a Kraken release, the PR is not "blocked" per se.  There are also other cases where we might put a PR on hold even though it is not actually blocked by anything else, although perhaps the concept of a draft PR covers this case.

The advantage of creating a more specific "Waiting for Kraken" label is that we can easily do a search for it and thus identify all PRs that should be merged when it is time to release the Kraken.

**Comment by hillaryj** [comment]
This sounds good - I propose adding two labels:
- `on hold` for things that are on hold for whatever reason but not blocked by something external
- `kraken` or `lineage-kraken` to indicate things that are kraken-ready

Examples:
- A PR that is on hold awaiting a coordinating PR would be labelled `on hold`
- A PR that is ready but awaiting the next Lineage kraken release would be labelled `on hold` `kraken`

**Comment by hillaryj** [comment]
My proposed labels (available in this repository):
<img width="749" alt="Screen Shot 2020-12-18 at 12 12 36 PM" src="https://user-images.githubusercontent.com/10235730/102641634-6262b500-412a-11eb-8b67-608e9a320622.png">

```
kraken: This pull request is ready to merge during the next Lineage kraken release
on hold: This pull request is awaiting an action like another pull request or a Lineage kraken release
```

**Comment by jsf9k** [comment]
> My proposed labels (available in this repository):
> <img alt="Screen Shot 2020-12-18 at 12 12 36 PM" width="749" src="https://user-images.githubusercontent.com/10235730/102641634-6262b500-412a-11eb-8b67-608e9a320622.png">
> 
> ```
> kraken: This pull request is ready to merge during the next Lineage kraken release
> on hold: This pull request is awaiting an action like another pull request or a Lineage kraken release
> ```

I don't think we should mention the Kraken in the description for the "on hold" label.  That just muddies the water, and in that case we might as well just overload the "on hold" label (which I don't want to do).

Also, recall that our naming of the Kraken is a reference to "Clash of the Titans", where the Kraken is a _particular_ beast and hence a proper noun; therefore, the word "Kraken" should be capitalized in the description for the "kraken" label.

**Comment by hillaryj** [comment]
How about:

```
on hold: This pull request is awaiting an action or decision to move forward
```

**Comment by dav3r** [comment]
I'm on board with adding the `on hold` and `kraken` labels.  I also agree with @jsf9k's comment that we should capitalize `Kraken` wherever possible.
